From cfe079ec466878a2540e4f1ec6028258c5cb2b1a Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Sat, 6 May 2023 11:39:47 -0400
Subject: [PATCH] expand r support

---
 DESCRIPTION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DESCRIPTION b/DESCRIPTION
index e115d4b..0c9ff59 100644
--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -16,7 +16,7 @@ Description: Classes and methods that allow the user to manage life table,
     actuarial tables (also multiple decrements tables). Moreover, functions to easily
     perform demographic, financial and actuarial mathematics on life contingencies
     insurances calculations are contained therein. See Spedicato (2013)	<doi:10.18637/jss.v055.i10>.
-Depends: R (>= 4.3), methods
+Depends: R (>= 4.1), methods
 Imports: parallel, utils, markovchain, Rcpp (>= 1.0.0), stats
 Suggests: demography, forecast, testthat, knitr, formatR, StMoMo,
         rmarkdown
